build.gradle.kts

```diff
@@ -139,6 +142,13 @@ gradlePlugin {
             implementationClass = "io.cloudflight.gradle.autoconfigure.swagger.SwaggerCodegenConfigurePlugin"
             tags.set(listOf("autoconfigure", "java", "kotlin"))
         }
+        create("springdoc-openapi-configure") {
+            id = "io.cloudflight.autoconfigure.springdoc-openapi-configure"
+            displayName = "Configure SpringDoc OpenApi Generation"
+            description = "Configure SpringDoc OpenApi Generation"
+            implementationClass = "io.cloudflight.gradle.autoconfigure.springdoc.openapi.SpringDocOpenApiConfigurePlugin"
+            tags.set(listOf("autoconfigure", "java", "kotlin"))
```

Comment:
So you want that line to be `tags.set(listOf("springdoc", "openapi"))`. Just asking because every plugin above has  exactly the same line setting the tags as `listOf("autoconfigure", "java", "kotlin")`.

Reply:
we should fix the others as well then (probably in a separate PR). the idea is to make plugins findable here: https://plugins.gradle.org/search?term=cloudflight